e2e/cephfs.go

```diff
@@ -76,6 +76,19 @@ func createORDeleteCephfsResources(action kubectlAction) {
 			e2elog.Failf("failed to %s CSIDriver object with error %v", action, err)
 		}
 	}
+	cephConf, err := ioutil.ReadFile(examplePath + cephConfconfigMap)
+	if err != nil {
+		// createORDeleteCephfsResources is used for upgrade testing as cephConfConfigmap is
+		// newly added, discarding file not found error.
```

Comment:
can we adjust or use a different method for upgrade testing, otherwise we will not be able to get the exact error if the file is really missing in new version ?

Reply:
I don't want to rework the framework on this PR and this is not something newly introduced this is following the existing model https://github.com/ceph/ceph-csi/blob/devel/e2e/rbd.go#L92-L104. if the file is missing then the pods won't come up and the pod describes will clearly mention that what is the issue.

Reply:
I doubt thats the case, if the pod fails due to missing configuration, its unlikcly the describe list the cause : ), are you sure on that ?

Reply:
I think there is a confusion, I was pointing it out the exact fil missing case and not really the volume mounted via manifest. but any way, considering this is in our e2e,  its fine..